.circleci/config.yml

```diff
@@ -64,6 +73,14 @@ jobs:
             cat .cache-hash
       - restore_cache:
           key: cache-{{ checksum ".cache-hash" }}
+      - run:
+          name: Install CMake and native library dependencies
+          command: |
+            sudo apt-get update && sudo apt-get install -yq cmake libicu-dev
+      - run:
+          name: Download Android SQLite
+          command: |
+            git clone https://android.googlesource.com/platform/external/sqlite $HOME/sqlite -b android11-release
```

Comment:
It is another place with old direct downloading without submodule.

Reply:
This file has been deleted from this PR.